seissolxdmfwriter/seissolxdmfwriter/seissolxdmfwriter.py

```diff
@@ -83,7 +147,20 @@ def write_seissol_output(prefix, xyz, connect, lDataName, lData, dt, lidt, reduc
     """
     nNodes = xyz.shape[0]
     nCells, node_per_element = connect.shape
-    write_seissol_xdmf(
-        prefix, nNodes, nCells, lDataName, lData, dt, node_per_element, lidt, reduce_precision
-    )
-    write_seissol_h5(prefix, lDataName, xyz, connect, lData, lidt, reduce_precision)
+    if len(lidt) == 1:
```

Comment:
Can we use instead 
```if  (dt == 0):```
I it may occur that there is only one item in lIdt but we still want to keep the time information.